composer.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "scriptfusion/static-class": "^1"
```

Comment:
I understand, however the dependency only does exactly what this use-case requires and nothing more, thus I think it is appropriate.